test/ui-test/src/screens/StatusWalletScreen.py

```diff
@@ -1,66 +1,282 @@
+import time
+import typing
 from ast import Tuple
 from enum import Enum
-import time
-import os
-import sys
-import common.Common as common
+
+import configs
+import constants
+import squish
 from common.SeedUtils import *
-from .StatusMainScreen import StatusMainScreen
-from .StatusMainScreen import authenticate_popup_enter_password
+from drivers import *
+from objectmaphelper import *
+from utils.ObjectAccess import *
+from utils.decorators import close_exists
+
 from .SettingsScreen import SidebarComponents
-from drivers.SquishDriver import type_text as type_text
+from .StatusMainScreen import authenticate_popup_enter_password
+from .components.base_popup import BasePopup
+from .components.confirmation_popup import ConfirmationPopup
+from .components.context_menu import ContextMenu
+from .components.remove_wallet_account_popup import RemoveWalletAccountPopup
+from .components.saved_address_popup import AddSavedAddressPopup, EditSavedAddressPopup
+from .components.wallet_account_popup import AccountPopup
 
 NOT_APPLICABLE = "N/A"
 VALUE_YES = "yes"
 VALUE_NO = "no"
 
+
+class MainWalletContextMenu(ContextMenu):
+
+    def __init__(self):
+        super(MainWalletContextMenu, self).__init__()
+        self._copy_address_menu_item = BaseElement('mainWallet_CopyAddress_MenuItem')
+        self._edit_account_menu_item = BaseElement('mainWallet_EditAccount_MenuItem')
+        self._add_new_account_menu_item = BaseElement('mainWallet_AddNewAccount_MenuItem')
+        self._add_watch_only_account_menu_item = BaseElement('mainWallet_AddWatchOnlyAccount_MenuItem')
+        self._delete_menu_item = BaseElement('mainWallet_DeleteAccount_MenuItem')
+
+    def wait_until_appears(self, timeout_msec: int = configs.squish.UI_LOAD_TIMEOUT_MSEC):
```

Comment:
Could it be a more generic name here? I don't find this name descriptive enough for the content it has!

```suggestion
    def wait_until_appears(self, timeout_msec: int = configs.timeouts.UI_LOAD_TIMEOUT_MSEC):
```

Reply:
> 
we just replace the wait_until_appears method from BaseElement here.
It is a common method for all objects.
But depending on the object content we can check that widget appears by waiting for the widget's elements.
For example, Popup have one locator for all (and we can not use it to check the exactly appears, which exact popup), but they have different content, so if one of the unique Popup's elements appears means that the popup opens.
I would save the name because it is common for all elements.

Reply:
I mean the `squish` file inside the `configs` not the `wait_until_appears` method!! I see that inside `squish` file, there are only timeouts defined!